packages/application-shell/src/components/navbar/navbar.tsx

```diff
@@ -213,13 +215,15 @@ const ApplicationMenu = (props: ApplicationMenuProps) => {
 ApplicationMenu.displayName = 'ApplicationMenu';
 
 type TNavbarProps = {
-  applicationLocale: string;
+  applicationLocale?: string;
```

Comment:
Just wondering why is not this property optional when it was required 🤔

Reply:
Thanks, that's a great point. Now the possibility of `undefined` is evaluated inside `NavBar`.
Type checking is extremely verbose 🙁:
https://github.com/commercetools/merchant-center-application-kit/blob/3004ea9b3d5441544f7f4337fbb5aadb32fb4c1d/packages/application-shell/src/components/navbar/navbar.tsx#L271-L284
but that's the way I could think of to narrow down the type without going for `applicationLocale={locale ?? ""}` in here:
https://github.com/commercetools/merchant-center-application-kit/blob/3004ea9b3d5441544f7f4337fbb5aadb32fb4c1d/packages/application-shell/src/components/application-shell-authenticated/application-shell-authenticated.tsx#L373
which could possibly change the behaviour (I think).